packages/dashboard/src/components/sidePanel/shared/expandableSectionHeader.tsx

```diff
@@ -14,7 +14,9 @@ const ExpandableSectionHeader: FC<PropsWithChildren<ExpandableSectionHeaderProps
       {children}
       {onClickButton && (
         <span className="expandable-section-header-icon">
-          <Icon name={iconName} />
+          <div onClick={onClickButton}>
+            <Icon name={iconName} />
```

Comment:
do icons not have an onclick property?

Reply:
No,`Icon` doesn't have an onclick property. I could use `<Button variant="icon" ... />` instead. But it would break some styles of the header.

Reply:
👍